pkgs/development/python-modules/withings-api/default.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchPypi
+, poetry
+, arrow
+, requests-oauth
+, requests-oauthlib
+, typing-extensions
+, pydantic
+}:
+
+buildPythonPackage rec {
+  pname = "withings-api";
+  version = "2.4.0";
+  format = "pyproject";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    pname = "withings_api"; # source and whl distribution use _ instead of -
```

```suggestion
    pname = "withings_api";
```

PEP503 normalization is pretty common, so we can do without the comment.